exporter/collector/monitoredresource.go

```diff
@@ -123,37 +130,37 @@ func (m *metricMapper) resourceMetricsToMonitoredResource(
 
 	switch cloudPlatform {
```

Comment:
I think @aabmass talked to you about this in standup.

I think you'll want resource-mapping config to allow altering this logic (the swtich on cloudPlatform and other related switches).

Reply:
Yep, thats what I ended up needing.  If you have feedback on the design, you can add it below.